<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>pluginSelector</class>
 <widget class="QWidget" name="pluginSelector">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="1">
    <spacer name="verticalSpacer_3">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>58</width>
       <height>113</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="1">
    <widget class="KPushButton" name="selectButton"/>
   </item>
   <item row="1" column="3">
    <widget class="KPushButton" name="upButton"/>
   </item>
   <item row="2" column="1">
    <widget class="KPushButton" name="deselectButton"/>
   </item>
   <item row="2" column="3">
    <widget class="KPushButton" name="downButton"/>
   </item>
   <item row="3" column="1">
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>58</width>
       <height>113</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="0" rowspan="4">
    <widget class="QTreeView" name="m_availableView"/>
   </item>
   <item row="0" column="2" rowspan="4">
    <widget class="QListView" name="m_selectedView"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KPushButton</class>
   <extends>QPushButton</extends>
   <header>kpushbutton.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
